src/Meshes.jl

```diff
@@ -294,6 +294,7 @@ export
   OverlappingBoxes,
   FaceTouchingBoxes,
   CornerTouchingBoxes,
+  IntersectingSegmentTri,
```

Comment:
I can't figure out what kind of intersection you have in mind from the name. I understand that the result can be a point, a segment or no intersection, correct?